rc/lf.kak

```diff
@@ -35,6 +36,17 @@ hook -group lf global GlobalSetOption 'lf_id=\d+' %{
     '
 }
 
+hook -group lf global GlobalSetOption 'lf_update_path=(true|false)' %{ evaluate-commands %sh{
+    case $kak_opt_lf_update_path in
+        true)
+            echo '
+            hook -group lf-update-path global WinDisplay .* %{
+                lf-send-command "select %val{buffile}"
```

Comment:
first you need to send cd command (user can open file from different directory)

Reply:
Sorry, force pushed, could you request again?